.github/workflows/pylinter.yml

```
@@ -0,0 +1,52 @@
+name: Pylint
+# ---------------------------------------#
+# ---------------------------------------#
+# ---   PYLINTER - Github Actions   -----#
+# ---------------------------------------#
+# ---------------------------------------#
+# Pylint for github actions. Runs seperate to superlinter
+
+# Documentation and source:
+# https://github.com/cclauss/GitHub-Action-for-pylint
+
+
+
+on:
+  push:
+    paths:
+      # Add specific files / filetypes here.
+      # Use `**` to wildcard through multiply paths
+      - "**.py"
+    # [un]-comment the following line to toggle for push to master
+    # branches-ignore: [master, main]
+
+  # This will only work when this workflow is on default branch
+  # TODO: Uncomment when merging
+  # pull_request:
+  #   branches: [dev, master, main]
```

Should these lines be uncommented now (and have updated branches)?
```suggestion
  pull_request:
    branches: [dev]
```

---

iirc, this needs to be done in two stages. The first (this one) gets the pipeline into dev and then a second quick one just to uncomment this

---

Hmm alright I'll trust that and we can do that afterwards.